staging/src/k8s.io/client-go/features/known_features.go

```diff
@@ -28,40 +28,40 @@ const (
 	// of code conflicts because changes are more likely to be scattered
 	// across the file.
 
-	// owner: @p0lyn0mial
-	// beta: v1.30
-	//
-	// Allow the client to get a stream of individual items instead of chunking from the server.
-	//
-	// NOTE:
-	//  The feature is disabled in Beta by default because
-	//  it will only be turned on for selected control plane component(s).
-	WatchListClient Feature = "WatchListClient"
-
-	// owner: @nilekhc
-	// alpha: v1.30
-	InformerResourceVersion Feature = "InformerResourceVersion"
-
 	// owner: @benluddy
 	// kep: https://kep.k8s.io/4222
+	// alpha: 1.32
 	//
 	// If disabled, clients configured to accept "application/cbor" will instead accept
 	// "application/json" with the same relative preference, and clients configured to write
 	// "application/cbor" or "application/apply-patch+cbor" will instead write
 	// "application/json" or "application/apply-patch+yaml", respectively.
-	//
-	// This feature is currently PRE-ALPHA and MUST NOT be enabled outside of integration tests.
-	TestOnlyClientAllowsCBOR Feature = "TestOnlyClientAllowsCBOR"
+	ClientsAllowCBOR Feature = "ClientsAllowCBOR"
 
 	// owner: @benluddy
 	// kep: https://kep.k8s.io/4222
+	// alpha: 1.32
 	//
-	// If enabled AND TestOnlyClientAllowsCBOR is also enabled, the default request content type
-	// (if not explicitly configured) and the dynamic client's request content type both become
-	// "application/cbor".
+	// If enabled, and only if ClientsAllowCBOR is also enabled, the default request content
+	// type (if not explicitly configured) and the dynamic client's request content type both
+	// become "application/cbor" instead of "application/json". The default content type for
+	// apply patch requests becomes "application/apply-patch+cbor" instead of
+	// "application/apply-patch+yaml".
+	ClientsPreferCBOR Feature = "ClientsPreferCBOR"
```

Comment:
This seems to cause the issues here https://github.com/kubernetes/kubernetes/issues/128600